src/MarginTrading.SettingsService.StorageInterfaces/IGenericCrudRepository.cs

```diff
@@ -1,17 +1,16 @@
 ﻿using System;
 using System.Collections.Generic;
 using System.Threading.Tasks;
-using Microsoft.WindowsAzure.Storage.Table;
 
-namespace MarginTrading.SettingsService.StorageInterfaces.Repositories
+namespace MarginTrading.SettingsService.StorageInterfaces
 {
     public interface IGenericCrudRepository<TD>
     {
         Task<IReadOnlyList<TD>> GetAsync();
         Task<IReadOnlyList<TD>> GetAsync(Func<TD, bool> filter);
-        Task<TD> GetAsync(string id);
+        Task<TD> GetAsync(string rowKey, string partitionKey = null);
```

Comment:
And how to get only by partition ID? 
I think method Task<IReadOnlyList<TD>> GetAsync(Func<TD, bool> filter); should include string partitionKey = null too.
It can be used in TradingInstrumentsController.AssignCollection